homeassistant/components/metoffice/weather.py

```diff
@@ -30,10 +41,8 @@ async def async_setup_entry(
 
     async_add_entities(
         [
-            MetOfficeWeather(
-                entry.data,
-                hass_data,
-            )
+            MetOfficeWeather(entry.data, hass_data, True),
+            MetOfficeWeather(entry.data, hass_data, False),
```

Comment:
Only one weather entity should be enabled by default for any integration. Please update the  corresponding entity property.

https://developers.home-assistant.io/docs/entity_registry_disabled_by#integrations-setting-default-value-of-disabled_by-for-new-entity-registry-entries